docs/release_process.md

```diff
@@ -46,26 +67,52 @@ This workflow will:
 - Run smoke tests
 - Create a draft release for both `ansible/awx` and `ansible/awx-operator`
 
-Once complete, navigate to the [Releases page](https://github.com/ansible/awx/releases) for AWX and verify things look ok. The changelog is automatically generated using the [special comment in our Pull Request template](https://github.com/ansible/awx/commit/dc0cc0f910900c506fb6f6ce4366e0e0d1d0ee87). If things look ok, click the pencil icon on the draft:
+## Promote the drag releases
```

Comment:
Do we need this section?

Reply:
I left it as a level 2 header as the theme of this section and then split out the releasing of AWX and Operator as level 3 headers under it. But we could promote the two level 3's back to level 2 to keep the document flater.